README.md

```diff
@@ -120,83 +120,108 @@ timekit.getConfig();
 
 ## Usage (endpoints)
 
-All the Timekit API endpoints are supported as methods. For endpoints taking parameters/data, the `data` argument should be an object with keys named as referenced in the docs - see: https://reference.timekit.io/reference/
+All the Timekit API endpoints are supported as methods. For endpoints taking parameters/data, the method argument should be an object with keys named as referenced in the docs - see: https://developers.timekit.io/reference/
 
-If you supply keys as camelCased, they will automatically be converted to snake_case for you. Responses can also be converted to camelCase automatically if you set the config variable "convertResponseToCamelcase" to true.
+If you supply keys as camelCased, they will automatically be converted to snake_case for you. Responses can also be converted to camelCase automatically if you set the config variable `convertResponseToCamelcase` to true.
 
 Endpoints/methods:
 
 ```javascript
 // App endpoints
-timekit.getApp(data);
+timekit.getApp()
 
 // Resource endpoints
-timekit.getResources();
-timekit.getResource(data);
-timekit.createResource(data);
-timekit.updateResource(data);
-timekit.resetResourcePassword(data);
-timekit.getResourceTimezone(data);
+timekit.getResources()
+timekit.getResource({ id })
+timekit.createResource({ ... })
+timekit.updateResource({ id, ... })
+timekit.resetResourcePassword({ ... })
+timekit.getResourceTimezone({ email })
 
 // FindTime endpoints
-timekit.findTime(data);
-timekit.findTimeBulk(data);
-timekit.findTimeTeam(data);
+timekit.findTime({ ... })
+timekit.findTimeBulk({ ... })
+timekit.findTimeTeam({ ... })
+
+// Availability endpoints
+timekit.fetchAvailability({ ... })
 
 // Booking endpoints
-timekit.getBookings();
-timekit.getBooking(data);
-timekit.createBooking(data);
-timekit.createBookingsBulk(data);
-timekit.updateBooking(data);
-timekit.updateBookingsBulk(data);
-timekit.getGroupBookings();
-timekit.getGroupBooking(data);
+timekit.getBookings()
+timekit.getBooking({ id })
+timekit.createBooking({ ... })
+timekit.createBookingsBulk({ ... })
+timekit.updateBooking({ id, action, ... })
+timekit.updateBookingsBulk({ ... })
+timekit.getGroupBookings()
+timekit.getGroupBooking({ id })
 
 // Auth endpoints (Note: only used to fetch a resource key)
-timekit.auth(data);
+timekit.auth({ ... })
 
 // Account endpoints
-timekit.getAccounts();
-timekit.accountGoogleSignup(data, shouldAutoRedirect:Boolean);
-timekit.accountSync(data);
+timekit.getAccounts()
+timekit.accountGoogleSignup({ callback }, shouldAutoRedirect)
```

Comment:
what about microsoft? does it use the same? should we consider renaming/adding?

Reply:
Ah, good point, I hadn't actually considered that. Let me check in the API routes